pubspec.yaml

```diff
@@ -8,7 +8,7 @@ environment:
 
 dependencies:
   _discoveryapis_commons: ^1.0.0
-  googleapis: ^3.0.0
+  googleapis: ^5.0.0
```

Comment:
If `package:gcloud` is compatible with both versions 3.x, 4.x and 5.x of googleapis, I suggest we write `googleapis: >=3.0.0 <6.0.0` that will make the migration much easier for most people.

Reply:
Good point!